src/main/java/ecommerce/model/Product.java

```diff
@@ -0,0 +1,49 @@
+package ecommerce.model;
+
+public class Product {
+    private long id;
```

Comment:
checklist says we should use wrapper Long for 'id' field

Reply:
done